### PR TITLE
Clean Up callingElement

### PR DESCRIPTION
**Pull request by stramel**
@justinfagnani Mind taking a look at this change? I think it simplifies things a bit.

### CODE REVIEW

File: lazy-imports-import.html [hunk 1]
```diff
@@ -19,35 +21,28 @@
...
-     *                                     resolving imports and fallback
-     *                                     for the dom-module if no
-     *                                     fromElment is passed.
-     * @param {string=} group The group attribute to filter on.
-     * @param {HTMLElement=} fromElement The element to search for lazy
+     * @param {HTMLElement} fromElement The element to search for lazy
      *                                   imports within. If no element is
      *                                   provided, this behavior will find a
-     *                                   `dom-module` with an id matching
+     *                                   `dom-module` with an `id` matching
```
Comment (justinfagnani) [inline lazy-imports-import.html]: An element has to always be provided now, yes? Also, it might make sense to allow a string id to be passed here too.

**Review by justinfagnani — CHANGES_REQUESTED** [review]
nice :)

File: lazy-imports-import.html (hunk 1 above)
Comment (stramel) [reply lazy-imports-import.html]: Yes, the internal function that does all the logic requires an element always be provided to it. The behavior and mixin both still have the optional `fromElement` which if not passed falls back to the calling element.

I could allow passing a string but how would I get the element reference to call `resolveUrl` and `importHref`?

My main thing was I haven't really been able to tell the difference between the `callingElement` and `fromElement` as it was previously.

Comment (stramel) [reply lazy-imports-import.html]: Need to update the documentation. I just realized you were pointing that out lol

**Pull request merged**